src/py42/clients/orgs.py

```diff
@@ -187,3 +408,14 @@ def get_agent_full_disk_access_states(self, guid):
                 :class:`py42.response.Py42Response`: A response containing settings information.
             """
         return self.get_agent_state(guid, u"fullDiskAccess")
+
+    def put_to_org_endpoint(self, org_id, data):
+        uri = "/api/Org/{}".format(org_id)
+        return self._session.put(uri, data=json.dumps(data))
+
+    def put_to_org_setting_endpoint(self, org_id, data):
```

Comment:
Are these `put` methods only used for settings? They seem quite generic. Should they maybe be internalized with underscores?

Reply:
Yeah, they're only used for modifying org settings. 

I guess the internalization question depends on how restrictive we want to be for end-users of py42 making any lower-level calls themselves. Do we only let them change what settings we've implemented (and if they _really_ need to they can use the internal `._session` object)? I'm more of the mind to say "here's the easy way to do it (using the OrgSettingsManager) or if you have something more you need to do manually, here's the basic low-level method".

Reply:
Maybe just renaming them to like `update_settings()`?

Reply:
I'm fine with these methods being exposed, but I think they need a more intuitive name. What does calling each of these endpoints _do_?

Reply:
The problem with `update_settings()` is that there are two endpoints that can do the "updating of settings", and there's no categorical distinction between which setting goes to which endpoint, apart from how they're stored in the server backend. 

So I kept it very low-level, where the function name is literally what it does, "make a put request to the OrgSetting endpoint", so that if that's what you want/need to do because the settings manager doesn't provide the functionality you need, you can craft your request data manually and make the put without having to dig into the internal session. 

I suppose we could have a single `update_settings()` function that takes two potential `data` arguments, like:

``
update_settings(
    org_id, 
    org_data={"some":"data"}, 
    org_settings_data={"some":"otherdata"}
)
```

Where if both types are passed in it makes requests to both endpoints?

Reply:
The problem with the current names is that it necessitates a knowledge of the HTTP verbs and url names and what their function is here. If I'm looking for a function that allow me to free-form update these settings and I don't know what endpoints are responsible for that (with py42 you shouldnt have to), these don't hit to me that they are what should be used.

I like your proposal since the name does indicate that. It's unfortunate that we will always have the problem of not knowing what the correct setting "destination" is, but that's still an improvement.

Reply:
Yeah, but the complexity of the data that you need to post to both of those endpoints is so high that knowing the verb and url name is inconsequential. 

I think maybe we should just restrict all py42 settings changes to the manager(s) and keep these private.

Reply:
I decided to remove the public PUT methods and just make all device/org settings changes go through the manager classes being passed to `update_org_settings()` and `update_device_settings()`.